server/opts.go

```diff
@@ -427,7 +436,7 @@ func (o *Options) ProcessConfigFile(configFile string) error {
 		case "disable_sublist_cache", "no_sublist_cache":
 			o.NoSublistCache = v.(bool)
 		case "accounts":
-			err := parseAccounts(tk, o, &errors, &warnings)
+			err := parseAccounts(tk, o, hasSystemAccount, &errors, &warnings)
```

Comment:
if we pass o and o is top level options, we can discover hasSystemAccount status inside yes?

Reply:
Right, if besides checking whether has system account we also set it at the beginning, would be possible to use the field from options, will update.